### PR TITLE
Add tail padding to the Emformer model.

### PR DESCRIPTION
**Pull request by csukuangfj**
Will try to port the Emformer model to the master branch in a separate PR.

### CODE REVIEW

File: egs/librispeech/ASR/transducer_emformer/decode.py [hunk 1]
```diff
@@ -247,13 +250,20 @@ def decode_one_batch(
     supervisions = batch["supervisions"]
     feature_lens = supervisions["num_frames"].to(device)
 
+    feature_lens += params.left_context_length
+    feature = torch.nn.functional.pad(
+        feature,
+        pad=(0, 0, 0, params.left_context_length),
+        value=LOG_EPS,
+    )
```
Comment (csukuangfj) [inline egs/librispeech/ASR/transducer_emformer/decode.py:258]: These lines add tail paddings.
